cmd/icinga-notifications-daemon/main.go

```diff
@@ -43,6 +51,7 @@ func main() {
 	}
 
 	logger := logs.GetLogger()
+	logger.Infof("Starting daemon with version: %s", internal.Version.Version)
```

Comment:
I think "with version" sounds a bit strange. I'd just go for "Starting Icinga Notifications daemon (v1.2.3-abcd)" (with or without parenthesis, no strong preference there)

Reply:
I prefer parenthesis.